ingest/src/main/java/com/microsoft/azure/kusto/ingest/utils/StreamUtils.java

```diff
@@ -0,0 +1,18 @@
+package com.microsoft.azure.kusto.ingest.utils;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class StreamUtils {
```

Comment:
Why the separate class (sounds like a resource manager capability)?

Reply:
Fixed